README.rst

```diff
@@ -104,20 +119,26 @@ The command returns (shell $? variable):
 - value 0 when the number of counted warnings is within the supplied minimum and maximum limits: ok,
 - number of counted warnings (positive) when the counter number is not within those limit.
 
-----------------------------
+-------------------------
 Parse for Sphinx warnings
-----------------------------
+-------------------------
 
 After you saved your Sphinx warnings to the file, you can parse it with
 command:
 
 .. code-block:: bash
 
-    # command line
+    # command line log file
     mlx-warnings doc_log.txt --sphinx
+    # command line command execution
+    mlx-warnings --command --sphinx commandforsphinx
```

Comment:
can you write <commandforsphinx> now it gives me the intention it is some kind of keyword...

Reply:
Fixed all around readme